projects/1.16.1/assets/tetra/tetra/lang/zh_cn.json

```
@@ -0,0 +1,2083 @@
+    "tetra.workbench.schematic_detail.craft": "合成",
+    "tetra.workbench.schematic_detail.back": "返回",
+    "tetra.workbench.schematic_detail.experience": "此操作需要 %d 级经验",
+    "tetra.workbench.schematic_list.temporary": "你的背包中有可用物品",
+    "tetra.repair_action.label": "修理",
+    "tetra.tool.hammer": "锤",
+    "tetra.tool.hammer.requirement": "此操作需要 %d 级的锤",
+    "tetra.tool.axe": "斧",
+    "tetra.tool.axe.requirement": "此操作需要 %d 级的斧",
+    "tetra.tool.pickaxe": "镐",
+    "tetra.tool.pickaxe.requirement": "此操作需要 %d 级的镐",
+    "tetra.tool.shovel": "锹",
+    "tetra.tool.shovel.requirement": "此操作需要 %d 级的锹",
+    "tetra.tool.cut": "切割",
+    "tetra.tool.cut.requirement": "这一步需要一个 %d 级的切割工具。\n§7大部分的剑可以被用于切割。",
+    "tetra.tool.pry": "撬",
+    "tetra.tool.pry.requirement": "这一步需要一个 %d 级的撬具。\n§7羊角头可以被用于撬。",
+    "tetra.tool.available": "你有一 个%d 级的",
+    "tetra.stats.efficiency_suffix": "效率",
+    "tetra.stats.strength_suffix": "力量",
+    "tetra.stats.tier_suffix": "等级",
+    "tetra.stats.level_suffix": "等级",
+    "tetra.stats.reach": "使用范围",
+    "tetra.stats.reach.tooltip": "可以触及最远 §e%s§r 格的方块和实体",
+    "tetra.stats.reach_normalized": "使用范围",
+    "tetra.stats.reach_normalized.tooltip": "增加这个物品的使用范围 §e%s§r 个方块",
+    "tetra.stats.durability": "耐久",
+    "tetra.stats.durability.tooltip": "这个物品有 §e%s§r 点耐久",
+    "tetra.stats.speed": "速度",
+    "tetra.stats.speed.tooltip": "影响物品可以被挥动得多快\n \n§e%s §7秒的挥动间隔\n§e%s§7 倍的采集效率",
+    "tetra.stats.speed.tooltip_extended": "挥动“冷却”由屏幕中央的指示条显示。可以在冷却完毕前挥动，但是攻击会造成更少的伤害。横扫或急速打击之类的效果只有在攻击指示条完全充满时才会被触发。§7速度值相当于采集效率的倍率。取决于大于或小于 1,它会增加或减少方块被破坏的速度。倍率计算方式如下：§f§o倍率 = 速度 * 0.5 + 0.5§r§7",
+    "tetra.stats.draw_speed": "拉弓耗时",
+    "tetra.stats.draw_speed.tooltip": "这个物品需要 §e%s §r秒来拉满",
+    "tetra.stats.draw_speed_normalized": "拉弓耗时",
+    "tetra.stats.draw_speed_normalized.tooltip": "将拉弓速度改变 §e%s 秒",
+    "tetra.stats.ability_speed": "冷却",
+    "tetra.stats.ability_speed.tooltip": "这个物品的特殊功能有 §e%s §r秒的基础冷却",
+    "tetra.stats.ability_speed_normalized": "冷却",
+    "tetra.stats.ability_speed_normalized.tooltip": "将基础冷却改变 §e%s §r秒",
+    "tetra.stats.speed_normalized": "速度",
+    "tetra.stats.speed_normalized.tooltip": "将攻击速度改变 §e%s",
+    "tetra.stats.attack_damage": "伤害",
+    "tetra.stats.attack_damage.tooltip": "造成 §e%s §r伤害",
+    "tetra.stats.attack_damage_normalized": "伤害",
+    "tetra.stats.attack_damage_normalized.tooltip": "将攻击伤害改变 §e%s 点",
+    "tetra.stats.draw_strength": "力量",
+    "tetra.stats.draw_strength.tooltip": "完全拉开时射出的弹射物会造成 §e%s - %s§r 的伤害，并具有每刻 §e%s§r 个方块的初速度",
+    "tetra.stats.draw_strength.tooltip_extended": "伤害计算基于原版的弹射物行为。伤害的下限取决于模块与升级，上限是：§f§o上限 = 下限 * 1.5 + 1§r§7。由不完全拉开的弓射出的弹射物伤害基于下限计算，例如：从 50%% 拉开的弓射出的弹射物会造成 50%% 的伤害下限。",
+    "tetra.stats.ability_damage": "伤害",
+    "tetra.stats.ability_damage.tooltip": "这个物品的特殊功能有 §e%s 点的基础伤害",
+    "tetra.stats.bleeding": "流血",
+    "tetra.stats.bleeding.tooltip": "攻击生物时有 30%% 的概率使它们以 §e%s§r 的伤害持续流血 2 秒",
```

|Feedback Priority|
|---|
|🌫 Dust / Non-blocking, “take it or leave it”|

```suggestion
    "tetra.stats.bleeding.tooltip": "攻击生物时有 30%% 的概率使它们以 §e%s§r 点的伤害持续流血 2 秒",
```